src/components/Header/Header.tsx

```diff
@@ -47,18 +51,25 @@ const Header: React.FC<Props> = ({ openSidebar, setOpenSidebar }) => {
       </UnstyledLink>
       <Navigation>
         <List>
-          {LINKS.map(({ href, name }) => (
-            <Item key={href} aria-selected={location.pathname === href}>
-              <Link
-                to={{
-                  pathname: href,
-                  search: location.search,
-                }}
-              >
-                {name}
-              </Link>
-            </Item>
-          ))}
+          {LINKS.map(({ href, name }) => {
+            const subRoute = parentRoutes.includes(href);
+            let ariaSelected = location.pathname === href;
```

Comment:
nit suggestion, does this work?
```suggestion
            const ariaSelected = location.pathname === href || (subRoute && location.pathname.includes(href));
```

Reply:
yep that does, cool.